src/code.cloudfoundry.org/nats-v2-migrate/natsinfo/nats_version.go

```diff
@@ -60,11 +60,13 @@ func GetMajorVersion(natsMachineUrl string) (int, error) {
 func connectWithRetry(natsMachineUrl string) (conn net.Conn, err error) {
 	attempts := int(NATSConnectionTimeout / NATSConnectionRetryInterval)
```

Comment:
I know this was here before, but I think it is confusing to calculate attempts instead of just having a variable. These variables aren't configurable right now. We should also make attempts a constant.